### ISSUE TITLE
More Metadata

### ISSUE THREAD

**Issue by GuiltyDolphin** [duckduckgo/community-platform#1349]
It would be really handy to have the following items provided in the metadata accessible via `DDG::Meta::Data` (through http://ddg-community.s3.amazonaws.com/metadata/repo_all.json.bz2).

1. CheatSheets

- [ ] The template (keyboard, language, etc.) used by the cheat sheet. We could perhaps have this in the field that other IAs use for style-template (text, info etc.)

2. Everything

- [ ] Locations of *all* back-end and front-end files.
- [ ] All example queries.

(Will update as I think of/require more).

/cc @MariagraziaAlastra @zachthompson

**Comment by zachthompson** [comment]
Yep, we have an task on the radar to get everything not directly related to handling the query in metadata.  Other things: triggers, the spice to/alt_to.  AFAIK queries are already feasible.

**Comment by GuiltyDolphin** [comment]
@zachthompson Yeah - we have the `example_query` already, but only the first one :cry:
